src/app/config/projectsJsonData.ts

```diff
@@ -72,4 +66,15 @@ export default class projectsJsonData{
       }
       return hosts;
     }
+
+    normalizeHostNameFromInput(input: string)
+    {
+      for (let key in this.m_projectJsonData.hostTypes)
```

Comment:
This function take "input" parameter which is matched against the known host key names, and the displayName is returned for the matching host. From that you could improve the naming of the function/parameter. For example: getHostDisplayName(hostKey: string).